src/components/MultiSelectInput/index.tsx

```diff
@@ -10,7 +11,9 @@ export type MultiSelectInputProps<
     // eslint-disable-next-line @typescript-eslint/ban-types
     OPTION extends object,
     RENDER_PROPS extends Def,
-> = SearchMultiSelectInputProps<OPTION_KEY, NAME, OPTION, RENDER_PROPS, 'onSearchValueChange' | 'searchOptions' | 'onShowDropdownChange' | 'totalOptionsCount'>;
+> = SearchMultiSelectInputProps<OPTION_KEY, NAME, OPTION, RENDER_PROPS, 'onSearchValueChange' | 'searchOptions' | 'onShowDropdownChange' | 'totalOptionsCount'> & {
+    showSelectAllButton?: boolean;
```

Comment:
We should remove onSelectAllButtonClick from the props for SelectInput, MultiSelectInput, SearchSelectInput.